docs/admin/researcher-setup/email-messaging.md

```diff
@@ -0,0 +1,139 @@
+
+### `service` configuration
+
+This section defines the number of events that will be sent by the service. Use the following table to configure options in the `service` section of the `configmap` file.
+
+|Component|Field|description|default|
+|:----|:----|:----|:----|
+|`service`|`service.concurrent_limit`|maximum number of events the service will handle in parallel|50|
+|`service`|`service.cached_events`|queue size for events before blocking the listener|1000|
+
+### `listener` configuration
+
+This section defines the objects and events that the service will listen to and send as notifications. Use the following table to configure options in the `listener` section of the `configmap` file.
+
+| Component | Field | description | default |
+| --- |  --- |  --- |  --- |
+| `kubelistener` | `listener.relevant_objects` | white list of Kubernetes components for notifications | relevant_objects: <br> `kind:` <br>    `Podreasons:UnschedulableScheduled` <br><br> `kind:` <br>`PodGroupreasons: -Evict` |
+| `kubelistener` | `listener.relevant_namespaces` | white list of namespaces that the service should listen to for events (regex) | `runai-.*` |
+
+### `enrich` configuration
+
+Use the following table to configure options in the `enrich` section of the `configmap` file.
+
+| Component | Field | description | default |
+| --- |  --- |  --- |  --- |
+|`KubeMetadata`|`enricher.kubeMetadata.lables`|maps the event related object labels to specific properties in the output event | `release: workloadDisplayName` <br><br>`training.kubeflow.org/job-name: workloadDisplayName` <br><br>`serving.knative.dev/service: workloadDisplayName` <br><br>`project: project`|
+|`KubeMetadata`|`enricher.kubeMetadata.annotations`|maps the related object annotations to event properties|`"user": "user"`|
+
+### `notify` configuration
+
+This section defines the notification configuration of the service and contains the details for the SMTP server and the recipients list.
+Use the following table to configure options in the `notify` section of the `configmap` file.
+
+|Component|Field|description|default|
+|--- |--- |--- |--- |
+|`Email`|`notify.email.smtp_host` (M)|SMTP server host address|Empty|
+|`Email`|`notify.email.smtp_port` (M)|SMTP server port|587|
+|`Email`|`notify.email.from_display_name` (M)|email's "From" display name|Run:ai|
+|`Email`|`notify.email.from` (M)|a valid domain source email address|test@run.ai|
+|`Email`|`notify.email.user` (M)|SMTP server user login|user|
+|`Email`|`notify.email.password` (M)|SMTP server user's password |password|
+|`Email`|`notify.email.direct_notifications` (together with Recipients)|when set to true, email notifications will be sent dynamically to the user who submitted the workload|false|
+|`Email`|`notify.email.recipients` (together with Direct Notifications)|additional email address recipients list for all the events - broadcast|Empty list|
+
+**(M)** = mandatory to include in the `configmap` file.
+
```

Comment:
```suggestion
You can change the service configuration values after installation, using the -f flag.
```
@jasonnovichRunAI I would add it here but we need to wait for Roi's answer as well

Reply:
ok